api/working-with-extensions/continuous-integration.md

```diff
@@ -197,6 +197,44 @@ In our example, the condition has three checks:
 - `startsWith( github.ref, 'refs/tags/releases/')` - Publish only if a tagged (release) build.
 - `matrix.os == 'ubuntu-latest'` - Include if your build runs on multiple agents (Windows, Linux, etc.). If not, remove that part of the condition.
 
+## GitLab CI
+
+GitLab CI can be used to test and publish the extension in headless Docker containers. This can be done by pulling a preconfigured Docker image, or installing `xvfb` and the libraries required to run Visual Studio Code during the pipeline.
+
+```yml
+image: node:12-buster
+
+before_script:
+  - npm install
+
+test:
+  script:
+    - apt update; apt install -y libasound2 libgbm-dev libgtk-3-0 libnss3 xvfb;
```

Comment:
This would be easier to read if [split](https://docs.gitlab.com/ee/ci/yaml/script.html#split-long-commands). `;` if executed in the shell will also not do what you want most likely